src/test/java/com/google/api/codegen/testdata/library_gapic.yaml

```diff
@@ -3,6 +3,8 @@ language_settings:
   java:
     package_name: com.google.gcloud.pubsub.spi
     release_level: BETA
+    class_names:
+      google.example.library.v1.LibraryService: LibraryClient
```

Comment:
It is for renaming from LibraryServiceClient->LibraryClient, and all other appropriate classes/variables.

I had hoped to allow for overriding the standard Client suffix as well, but you are right that this change must apply to the settings classes, and test class. Have updated accordingly.